source/framework/vx_graph_verify.c

```diff
@@ -730,98 +727,189 @@ static vx_status ownGraphCalcInAndOutNodes(vx_graph graph)
     vx_reference ref1, ref2;
     vx_status status = (vx_status)VX_SUCCESS;
 
-    for(node_cur_idx=0; node_cur_idx<graph->num_nodes; node_cur_idx++)
+    for (node_cur_idx = 0; node_cur_idx < graph->num_nodes && VX_SUCCESS == status; node_cur_idx++)
     {
         node_cur = graph->nodes[node_cur_idx];
-
-        for(prm_cur_idx=0; prm_cur_idx<ownNodeGetNumParameters(node_cur); prm_cur_idx++)
+        uint32_t num_node_params = ownNodeGetNumParameters(node_cur);
+        if (16 < num_node_params)
```

Comment:
Can you use a #define for the value of 16 rather than hardcoding it?

Reply:
The value we have that already exists for this is "TIVX_KERNEL_MAX_PARAMS", but it is equal to 64.  Is there a reason this needs to be 16 or can it use the TIVX_KERNEL_MAX_PARAMS value?

Reply:
the max of parameter for the standard nodes is currently 11, in addition the the bitmask is 32bits, so you cannot go above.  TIDL can go above this limit, to be check @TI. Bosch modified the TIDL implementation to store the output tensor in a vx_array.

Reply:
After internal review, we have changed the TIVX_KERNEL_MAX_PARAMS to 32.  Therefore, can we then update this portion of code to use this macro?

Reply:
done